components/eamxx/src/control/atmosphere_surface_coupling_exporter.hpp

```diff
@@ -69,7 +77,10 @@ class SurfaceCouplingExporter : public AtmosphereProcess
   // If calling in initialize_impl(), set
   // called_during_initialization=true to avoid exporting fields
   // which do not have valid entries.
-  void do_export(const double dt, const bool called_during_initialization=false);
+  void do_export(const double dt, const bool called_during_initialization=false);             // Main export routine
+  void do_export_from_eamxx(const double dt, const bool called_during_initialization=false);  // Export vars are derived from eamxx state
+  void do_export_constant(const double dt, const bool called_during_initialization=false);    // Export vars are set to a constant
+  void do_export_to_cpl(const bool called_during_initialization=false);                       // Finish export by copying data to cpl structures.
```

Comment:
For naming, I think something like `compute_eamxx_exports()` `set_constant_exports()` `do_export_to_cpl()` would be better, but feel free to ignore.

Reply:
Agreed.